Revise proposal and voting process documentation

lrettig opened the pull request:
Updated the proposal and voting process with detailed steps and clarifications

Copying over content from https://hackmd.io/Uk5lFBqvTI6hxHyZrOf_UQ

lrettig committed:
Revise proposal and voting process documentation

Updated the proposal and voting process with detailed steps and clarifications

tksohishi commented on docs/governance-system/proposal-and-voting-process.md:
```diff
@@ -1,18 +1,24 @@
-</a>
-<!-- prettier-ignore-end -->
+3. Draft proposal submitted to [proposals repository](https://github.com/houseofstake/proposals) as a PR that creates a new HSP in markdown format. See [sample PR](https://github.com/houseofstake/proposals/pull/16). PR target filename should be `HSPs/hsp-xxx.md` in the initial PR, as a HSP number has not yet been assigned. 
+4. HSP editor reviews proposal draft and make sure it conforms to the spec: correct format, complete (contains all required sections), and is within the scope of the HSP process/House of Stake governance. HSP editor communicates with author if changes are required.
```
Simply, Screening Committee member?

AngelaKTE replied:
> To be clear, the two roles are quite distinct and the demands are distinct. The HSP editor needs to be someone who's extremely conscientious, and who just reviews proposals for formatting, completeness, consistency, etc. The HSP editor is not expressing an opinion on the content of the proposal. By contrast, the screening committee should block/refuse to approve proposals on the grounds that they disagree with the content.

Agreed!

tksohishi commented on docs/governance-system/proposal-and-voting-process.md:
```diff
@@ -1,18 +1,24 @@
+
+- A HSP author may, at any time, withdraw a proposal, changing its status on Github to "Withdrawn."
+- A HSP editor may mark a HSP as "Stagnant" once it's been inactive for six months. "Stagnant" proposals are not eligible for a vote, but the author can ressurrect a "Stagnant" proposal by moving it back to "Draft" at any time.
+- Another party may pick up and choose to champion a proposal if the authors withdraw it, or if it becomes stagnant. In this case, they become the new Author.
```
It's simpler and less confusing not to introduce to reviving the proposal

tksohishi reviewed (commented):
Made a few comments extending my comments in the hackmd version

lrettig committed:
Update terminology from 'temperature check' to 'feedback period'

cf https://gov.near.org/t/ratifying-our-constitutional-documents-upcoming-proposals-dec-2025/41808/8

dancunningham commented on docs/governance-system/proposal-and-voting-process.md:
```diff
@@ -1,18 +1,24 @@
-Contribute
-</a>
-<!-- prettier-ignore-end -->
+3. Draft proposal submitted to [proposals repository](https://github.com/houseofstake/proposals) as a PR that creates a new HSP in markdown format. See [sample PR](https://github.com/houseofstake/proposals/pull/16). PR target filename should be `HSPs/hsp-xxx.md` in the initial PR, as a HSP number has not yet been assigned. 
```
```suggestion
3. Draft proposal submitted to [proposals repository](https://github.com/houseofstake/proposals) as a PR that creates a new HSP in markdown format. See [sample PR](https://github.com/houseofstake/proposals/pull/18). PR target filename should be `HSPs/hsp-xxx.md` in the initial PR, as a HSP number has not yet been assigned. Here's a direct link to create a new file in the correct folder: https://github.com/houseofstake/proposals/new/main/HSPs
```

dancunningham commented on docs/governance-system/proposal-and-voting-process.md:
```diff
@@ -1,18 +1,24 @@
-<!-- prettier-ignore-end -->
+3. Draft proposal submitted to [proposals repository](https://github.com/houseofstake/proposals) as a PR that creates a new HSP in markdown format. See [sample PR](https://github.com/houseofstake/proposals/pull/16). PR target filename should be `HSPs/hsp-xxx.md` in the initial PR, as a HSP number has not yet been assigned. 
+4. HSP editor reviews proposal draft and make sure it conforms to the spec: correct format, complete (contains all required sections), and is within the scope of the HSP process/House of Stake governance. HSP editor communicates with author if changes are required.
+5. Once HSP conforms to the spec, an HSP editor officially assigns it a HSP ID number, assigns it the "Draft" status, and merges it. At this point, the proposal is a canonical HSP and is eligible for voting (once the feedback period ends).
```
To keep status transitions really clear, I suggest that this action, which corresponds with when the proposal gets its HSP number, should also trigger the transition from Draft to Review, as it means the proposal is correctly formatted and complete.

dancunningham commented on docs/governance-system/proposal-and-voting-process.md:
```diff
@@ -1,18 +1,24 @@
+3. Draft proposal submitted to [proposals repository](https://github.com/houseofstake/proposals) as a PR that creates a new HSP in markdown format. See [sample PR](https://github.com/houseofstake/proposals/pull/16). PR target filename should be `HSPs/hsp-xxx.md` in the initial PR, as a HSP number has not yet been assigned. 
+4. HSP editor reviews proposal draft and make sure it conforms to the spec: correct format, complete (contains all required sections), and is within the scope of the HSP process/House of Stake governance. HSP editor communicates with author if changes are required.
+5. Once HSP conforms to the spec, an HSP editor officially assigns it a HSP ID number, assigns it the "Draft" status, and merges it. At this point, the proposal is a canonical HSP and is eligible for voting (once the temperature check period ends).
+6. Any author can set the HSP status to "Review", "Last Call", or "Voting." It's suggested, but not required, that this process continue for at least two weeks before the "Voting" status is assigned. HSP editors should promptly merge such changes.
```
I think we should remove this line for a few reasons:

1. We've already said this is part of the "7 day feedback period" so it's confusing to have a suggestion for a 2+ week process within that
2. As discussed, there's no clear distinction between "Review" and "Last Call" so we can drop "Last Call"
3. I believe it would be clearer if the transition to "Voting" corresponds to when the Screening Committee approves the proposal to go to vote. That way, it would be a Screening Committee member or HSP editor who would edit the status to "Voting" and that would correspond to when people can actually start voting on it.

dancunningham commented on docs/governance-system/proposal-and-voting-process.md:
```diff
@@ -1,18 +1,24 @@
+- A HSP author may, at any time, withdraw a proposal, changing its status on Github to "Withdrawn."
+- A HSP editor may mark a HSP as "Stagnant" once it's been inactive for six months. "Stagnant" proposals are not eligible for a vote, but the author can ressurrect a "Stagnant" proposal by moving it back to "Draft" at any time.
+- Another party may pick up and choose to champion a proposal if the authors withdraw it, or if it becomes stagnant. In this case, they become the new Author.
+- By the same token, any proposal may be forked by anyone at any time, permissionlessly. Note that proposals are all released into the public domain via [CC0 License](https://creativecommons.org/publicdomain/zero/1.0/).
```
Make it clear that this would create a new proposal with a new HSP number

dancunningham reviewed (changes requested):
It's very close to being a good enough starting point and creating much-needed clarify.

Just a few suggestions to consider to make things even clearer, thanks!

dancunningham commented:
@lrettig after these suggestions are considered and acted on, I think you can mark this as Ready for review so we can get it approved and get it live ASAP. Even if there are further refinements we want to make, having something up that clearly describes the as-is process and is not wrong is a good starting point!

AngelaKTE commented:
## HSP-001 update request to match neargov.ai
The current version of https://neargov.ai/proposals includes a required “KPIs” section.

I recommend adding this section to HSP-001/Voting Process docs because:
- It encourages proposers to think about how positive impact will be measured.
- It enables delegates to verify the feasibility and maturity of the impact claims before voting.
- It allows HoS to evaluate deliverables against clear and transparent metrics.

In some cases, KPIs may be difficult to define. In those situations, the existing guidance on optional sections applies: “Proposals may omit sections that are not applicable, but should justify their omission. Additional sections may be added if they improve clarity.”

dancunningham committed:
Update to use an approved PR as the example rather than a closed one

lrettig committed:
Update docs/governance-system/proposal-and-voting-process.md

Co-authored-by: Dan Cunningham <daniel.cunningham@gmail.com>

lrettig committed:
Fix formatting and clarify HSP status steps

lrettig committed:
Add submission link for House of Stake voting

Updated the process for submitting proposals to include a direct link for submission to House of Stake.

lrettig reviewed (commented):
incorporated feedback and resolved all outstanding comments
